### DISCUSSION TITLE
Github Actions npm install fails without changes to package.json

### DISCUSSION THREAD

**Discussion by shroomlife** [community/community#25794] [Actions]
<p>Hey,</p>
<p>I use Github actions for my repository to deploy to the actual running server. This works great until yesterday. I just commited an update and pushed it. The Action got triggered but then there was a failure. The failure came from the following command:</p>
<pre><code class="lang-auto">npm install
</code></pre>
<p>I have no idea why this is happening since I didn’t change anthing about configuration or even the package.json file. My project is dependant to the sass library and this seems to create the following errors:</p>
<pre><code class="lang-auto">npm ERR! code 1
npm ERR! path /home/runner/work/***/***/node_modules/node-sass
npm ERR! command failed
npm ERR! command sh -c node scripts/build.js
npm ERR! Building: /usr/local/bin/node /home/runner/work/***/***/node_modules/node-gyp/bin/node-gyp.js rebuild --verbose --libsass_ext= --libsass_cflags= --libsass_ldflags= --libsass_library=
npm ERR! make: Entering directory '/home/runner/work/***/***/node_modules/node-sass/build'
npm ERR!   g++ '-DNODE_GYP_MODULE_NAME=libsass' '-DUSING_UV_SHARED=1' '-DUSING_V8_SHARED=1' '-DV8_DEPRECATION_WARNINGS=1' '-DV8_DEPRECATION_WARNINGS' '-DV8_IMMINENT_DEPRECATION_WARNINGS' '-D_GLIBCXX_USE_CXX11_ABI=1' '-D_LARGEFILE_SOURCE' '-D_FILE_OFFSET_BITS=64' '-D__STDC_FORMAT_MACROS' '-DOPENSSL_NO_PINSHARED' '-DOPENSSL_THREADS' '-DLIBSASS_VERSION="3.5.5"' -I/home/runner/.node-gyp/16.13.0/include/node -I/home/runner/.node-gyp/16.13.0/src -I/home/runner/.node-gyp/16.13.0/deps/openssl/config -I/home/runner/.node-gyp/16.13.0/deps/openssl/openssl/include -I/home/runner/.node-gyp/16.13.0/deps/uv/include -I/home/runner/.node-gyp/16.13.0/deps/zlib -I/home/runner/.node-gyp/16.13.0/deps/v8/include -I../src/libsass/include  -fPIC -pthread -Wall -Wextra -Wno-unused-parameter -m64 -O3 -fno-omit-frame-pointer -std=gnu++14 -std=c++0x -fexceptions -frtti -MMD -MF ./Release/.deps/Release/obj.target/libsass/src/libsass/src/ast.o.d.raw   -c -o Release/obj.target/libsass/src/libsass/src/ast.o ../src/libsass/src/ast.cpp
npm ERR!   g++ '-DNODE_GYP_MODULE_NAME=libsass' '-DUSING_UV_SHARED=1' '-DUSING_V8_SHARED=1' '-DV8_DEPRECATION_WARNINGS=1' '-DV8_DEPRECATION_WARNINGS' '-DV8_IMMINENT_DEPRECATION_WARNINGS' '-D_GLIBCXX_USE_CXX11_ABI=1' '-D_LARGEFILE_SOURCE' '-D_FILE_OFFSET_BITS=64' '-D__STDC_FORMAT_MACROS' '-DOPENSSL_NO_PINSHARED' '-DOPENSSL_THREADS' '-DLIBSASS_VERSION="3.5.5"' -I/home/runner/.node-gyp/16.13.0/include/node -I/home/runner/.node-gyp/16.13.0/src -I/home/runner/.node-gyp/16.13.0/deps/openssl/config -I/home/runner/.node-gyp/16.13.0/deps/openssl/openssl/include -I/home/runner/.node-gyp/16.13.0/deps/uv/include -I/home/runner/.node-gyp/16.13.0/deps/zlib -I/home/runner/.node-gyp/16.13.0/deps/v8/include -I../src/libsass/include  -fPIC -pthread -Wall -Wextra -Wno-unused-parameter -m64 -O3 -fno-omit-frame-pointer -std=gnu++14 -std=c++0x -fexceptions -frtti -MMD -MF ./Release/.deps/Release/obj.target/libsass/src/libsass/src/ast_fwd_decl.o.d.raw   -c -o Release/obj.target/libsass/src/libsass/src/ast_fwd_decl.o ../src/libsass/src/ast_fwd_decl.cpp

...

npm ERR!   rm -f Release/obj.target/src/sass.a &amp;&amp; ar crs Release/obj.target/src/sass.a Release/obj.target/libsass/src/libsass/src/ast.o Release/obj.target/libsass/src/libsass/src/ast_fwd_decl.o Release/obj.target/libsass/src/libsass/src/backtrace.o Release/obj.target/libsass/src/libsass/src/base64vlq.o Release/obj.target/libsass/src/libsass/src/bind.o Release/obj.target/libsass/src/libsass/src/cencode.o Release/obj.target/libsass/src/libsass/src/check_nesting.o Release/obj.target/libsass/src/libsass/src/color_maps.o Release/obj.target/libsass/src/libsass/src/constants.o Release/obj.target/libsass/src/libsass/src/context.o Release/obj.target/libsass/src/libsass/src/cssize.o Release/obj.target/libsass/src/libsass/src/emitter.o Release/obj.target/libsass/src/libsass/src/environment.o Release/obj.target/libsass/src/libsass/src/error_handling.o Release/obj.target/libsass/src/libsass/src/eval.o Release/obj.target/libsass/src/libsass/src/expand.o Release/obj.target/libsass/src/libsass/src/extend.o Release/obj.target/libsass/src/libsass/src/file.o Release/obj.target/libsass/src/libsass/src/functions.o Release/obj.target/libsass/src/libsass/src/inspect.o Release/obj.target/libsass/src/libsass/src/json.o Release/obj.target/libsass/src/libsass/src/lexer.o Release/obj.target/libsass/src/libsass/src/listize.o Release/obj.target/libsass/src/libsass/src/memory/SharedPtr.o Release/obj.target/libsass/src/libsass/src/node.o Release/obj.target/libsass/src/libsass/src/operators.o Release/obj.target/libsass/src/libsass/src/output.o Release/obj.target/libsass/src/libsass/src/parser.o Release/obj.target/libsass/src/libsass/src/plugins.o Release/obj.target/libsass/src/libsass/src/position.o Release/obj.target/libsass/src/libsass/src/prelexer.o Release/obj.target/libsass/src/libsass/src/remove_placeholders.o Release/obj.target/libsass/src/libsass/src/sass.o Release/obj.target/libsass/src/libsass/src/sass2scss.o Release/obj.target/libsass/src/libsass/src/sass_context.o Release/obj.target/libsass/src/libsass/src/sass_functions.o Release/obj.target/libsass/src/libsass/src/sass_util.o Release/obj.target/libsass/src/libsass/src/sass_values.o Release/obj.target/libsass/src/libsass/src/source_map.o Release/obj.target/libsass/src/libsass/src/subset_map.o Release/obj.target/libsass/src/libsass/src/to_c.o Release/obj.target/libsass/src/libsass/src/to_value.o Release/obj.target/libsass/src/libsass/src/units.o Release/obj.target/libsass/src/libsass/src/utf8_string.o Release/obj.target/libsass/src/libsass/src/util.o Release/obj.target/libsass/src/libsass/src/values.o
npm ERR!   rm -rf "Release/sass.a" &amp;&amp; cp -af "Release/obj.target/src/sass.a" "Release/sass.a"
npm ERR!   g++ '-DNODE_GYP_MODULE_NAME=binding' '-DUSING_UV_SHARED=1' '-DUSING_V8_SHARED=1' '-DV8_DEPRECATION_WARNINGS=1' '-DV8_DEPRECATION_WARNINGS' '-DV8_IMMINENT_DEPRECATION_WARNINGS' '-D_GLIBCXX_USE_CXX11_ABI=1' '-D_LARGEFILE_SOURCE' '-D_FILE_OFFSET_BITS=64' '-D__STDC_FORMAT_MACROS' '-DOPENSSL_NO_PINSHARED' '-DOPENSSL_THREADS' '-DBUILDING_NODE_EXTENSION' -I/home/runner/.node-gyp/16.13.0/include/node -I/home/runner/.node-gyp/16.13.0/src -I/home/runner/.node-gyp/16.13.0/deps/openssl/config -I/home/runner/.node-gyp/16.13.0/deps/openssl/openssl/include -I/home/runner/.node-gyp/16.13.0/deps/uv/include -I/home/runner/.node-gyp/16.13.0/deps/zlib -I/home/runner/.node-gyp/16.13.0/deps/v8/include -I../../nan -I../src/libsass/include  -fPIC -pthread -Wall -Wextra -Wno-unused-parameter -m64 -O3 -fno-omit-frame-pointer -fno-rtti -fno-exceptions -std=gnu++14 -std=c++0x -MMD -MF ./Release/.deps/Release/obj.target/binding/src/binding.o.d.raw   -c -o Release/obj.target/binding/src/binding.o ../src/binding.cpp
npm ERR! binding.target.mk:133: recipe for target 'Release/obj.target/binding/src/binding.o' failed
npm ERR! make: Leaving directory '/home/runner/work/***/***/node_modules/node-sass/build'
npm ERR! gyp info it worked if it ends with ok
npm ERR! gyp verb cli [
npm ERR! gyp verb cli   '/usr/local/bin/node',
npm ERR! gyp verb cli   '/home/runner/work/***/***/node_modules/node-gyp/bin/node-gyp.js',
npm ERR! gyp verb cli   'rebuild',
npm ERR! gyp verb cli   '--verbose',
npm ERR! gyp verb cli   '--libsass_ext=',
npm ERR! gyp verb cli   '--libsass_cflags=',
npm ERR! gyp verb cli   '--libsass_ldflags=',
npm ERR! gyp verb cli   '--libsass_library='
npm ERR! gyp verb cli ]
npm ERR! gyp info using node-gyp@3.8.0
npm ERR! gyp info using node@16.13.0 | linux | x64
npm ERR! gyp verb command rebuild []
npm ERR! gyp verb command clean []
npm ERR! gyp verb clean removing "build" directory
npm ERR! gyp verb command configure []
npm ERR! gyp verb check python checking for Python executable "python2" in the PATH
npm ERR! gyp verb `which` succeeded python2 /usr/bin/python2
npm ERR! gyp verb check python version `/usr/bin/python2 -c "import sys; print "2.7.17
npm ERR! gyp verb check python version .%s.%s" % sys.version_info[:3];"` returned: %j
npm ERR! gyp verb get node dir no --target version specified, falling back to host node version: 16.13.0
npm ERR! gyp verb command install [ '16.13.0' ]
npm ERR! gyp verb install input version string "16.13.0"
npm ERR! gyp verb install installing version: 16.13.0
npm ERR! gyp verb install --ensure was passed, so won't reinstall if already installed
npm ERR! gyp verb install version is already installed, need to check "installVersion"
npm ERR! gyp verb got "installVersion" 9
npm ERR! gyp verb needs "installVersion" 9
npm ERR! gyp verb install version is good
npm ERR! gyp verb get node dir target node version installed: 16.13.0
npm ERR! gyp verb build dir attempting to create "build" dir: /home/runner/work/***/***/node_modules/node-sass/build
npm ERR! gyp verb build dir "build" dir needed to be created? /home/runner/work/***/***/node_modules/node-sass/build
npm ERR! gyp verb build/config.gypi creating config file
npm ERR! gyp verb build/config.gypi writing out config file: /home/runner/work/***/***/node_modules/node-sass/build/config.gypi
npm ERR! (node:3585) [DEP0150] DeprecationWarning: Setting process.config is deprecated. In the future the property will be read-only.
npm ERR! (Use `node --trace-deprecation ...` to show where the warning was created)
npm ERR! gyp verb config.gypi checking for gypi file: /home/runner/work/***/***/node_modules/node-sass/config.gypi
npm ERR! gyp verb common.gypi checking for gypi file: /home/runner/work/***/***/node_modules/node-sass/common.gypi
npm ERR! gyp verb gyp gyp format was not specified; forcing "make"
npm ERR! gyp info spawn /usr/bin/python2
npm ERR! gyp info spawn args [
npm ERR! gyp info spawn args   '/home/runner/work/***/***/node_modules/node-gyp/gyp/gyp_main.py',
npm ERR! gyp info spawn args   'binding.gyp',
npm ERR! gyp info spawn args   '-f',
npm ERR! gyp info spawn args   'make',
npm ERR! gyp info spawn args   '-I',
npm ERR! gyp info spawn args   '/home/runner/work/***/***/node_modules/node-sass/build/config.gypi',
npm ERR! gyp info spawn args   '-I',
npm ERR! gyp info spawn args   '/home/runner/work/***/***/node_modules/node-gyp/addon.gypi',
npm ERR! gyp info spawn args   '-I',
npm ERR! gyp info spawn args   '/home/runner/.node-gyp/16.13.0/include/node/common.gypi',
npm ERR! gyp info spawn args   '-Dlibrary=shared_library',
npm ERR! gyp info spawn args   '-Dvisibility=default',
npm ERR! gyp info spawn args   '-Dnode_root_dir=/home/runner/.node-gyp/16.13.0',
npm ERR! gyp info spawn args   '-Dnode_gyp_dir=/home/runner/work/***/***/node_modules/node-gyp',
npm ERR! gyp info spawn args   '-Dnode_lib_file=/home/runner/.node-gyp/16.13.0/&lt;(target_arch)/node.lib',
npm ERR! gyp info spawn args   '-Dmodule_root_dir=/home/runner/work/***/***/node_modules/node-sass',
npm ERR! gyp info spawn args   '-Dnode_engine=v8',
npm ERR! gyp info spawn args   '--depth=.',
npm ERR! gyp info spawn args   '--no-parallel',
npm ERR! gyp info spawn args   '--generator-output',
npm ERR! gyp info spawn args   'build',
npm ERR! gyp info spawn args   '-Goutput_dir=.'
npm ERR! gyp info spawn args ]
npm ERR! gyp verb command build []
npm ERR! gyp verb build type Release
npm ERR! gyp verb architecture x64
npm ERR! gyp verb node dev dir /home/runner/.node-gyp/16.13.0
npm ERR! gyp verb `which` succeeded for `make` /usr/bin/make
npm ERR! gyp info spawn make
npm ERR! gyp info spawn args [ 'V=1', 'BUILDTYPE=Release', '-C', 'build' ]
npm ERR! ../src/libsass/src/cencode.c: In function ‘base64_encode_block’:
npm ERR! ../src/libsass/src/cencode.c:48:11: warning: this statement may fall through [-Wimplicit-fallthrough=]
npm ERR!     result = (fragment &amp; 0x003) &lt;&lt; 4;
npm ERR!     ~~~~~~~^~~~~~~~~~~~~~~~~~~~~~~~~
npm ERR! ../src/libsass/src/cencode.c:52:2: note: here
npm ERR!   case step_B:
npm ERR!   ^~~~
npm ERR! ../src/libsass/src/cencode.c:62:11: warning: this statement may fall through [-Wimplicit-fallthrough=]
npm ERR!     result = (fragment &amp; 0x00f) &lt;&lt; 2;
npm ERR!     ~~~~~~~^~~~~~~~~~~~~~~~~~~~~~~~~
npm ERR! ../src/libsass/src/cencode.c:66:2: note: here
npm ERR!   case step_C:
npm ERR!   ^~~~
npm ERR! In file included from /home/runner/.node-gyp/16.13.0/include/node/v8.h:30:0,
npm ERR!                  from /home/runner/.node-gyp/16.13.0/include/node/node.h:63,
npm ERR!                  from ../../nan/nan.h:58,
npm ERR!                  from ../src/binding.cpp:1:
npm ERR! /home/runner/.node-gyp/16.13.0/include/node/v8-internal.h: In function ‘void v8::internal::PerformCastCheck(T*)’:
npm ERR! /home/runner/.node-gyp/16.13.0/include/node/v8-internal.h:492:38: error: ‘remove_cv_t’ is not a member of ‘std’
npm ERR!              !std::is_same&lt;Data, std::remove_cv_t&lt;T&gt;&gt;::value&gt;::Perform(data);
npm ERR!                                       ^~~~~~~~~~~
npm ERR! /home/runner/.node-gyp/16.13.0/include/node/v8-internal.h:492:38: note: suggested alternative: ‘remove_cv’
npm ERR!              !std::is_same&lt;Data, std::remove_cv_t&lt;T&gt;&gt;::value&gt;::Perform(data);
npm ERR!                                       ^~~~~~~~~~~
npm ERR!                                       remove_cv
npm ERR! /home/runner/.node-gyp/16.13.0/include/node/v8-internal.h:492:38: error: ‘remove_cv_t’ is not a member of ‘std’
npm ERR! /home/runner/.node-gyp/16.13.0/include/node/v8-internal.h:492:38: note: suggested alternative: ‘remove_cv’
npm ERR!              !std::is_same&lt;Data, std::remove_cv_t&lt;T&gt;&gt;::value&gt;::Perform(data);
npm ERR!                                       ^~~~~~~~~~~
npm ERR!                                       remove_cv
npm ERR! /home/runner/.node-gyp/16.13.0/include/node/v8-internal.h:492:50: error: template argument 2 is invalid
npm ERR!              !std::is_same&lt;Data, std::remove_cv_t&lt;T&gt;&gt;::value&gt;::Perform(data);
npm ERR!                                                   ^
npm ERR! /home/runner/.node-gyp/16.13.0/include/node/v8-internal.h:492:63: error: ‘::Perform’ has not been declared
npm ERR!              !std::is_same&lt;Data, std::remove_cv_t&lt;T&gt;&gt;::value&gt;::Perform(data);
npm ERR!                                                                ^~~~~~~
npm ERR! /home/runner/.node-gyp/16.13.0/include/node/v8-internal.h:492:63: note: suggested alternative: ‘herror’
npm ERR!              !std::is_same&lt;Data, std::remove_cv_t&lt;T&gt;&gt;::value&gt;::Perform(data);
npm ERR!                                                                ^~~~~~~
npm ERR!                                                                herror
npm ERR! make: *** [Release/obj.target/binding/src/binding.o] Error 1
npm ERR! gyp ERR! build error 
npm ERR! gyp ERR! stack Error: `make` failed with exit code: 2
npm ERR! gyp ERR! stack     at ChildProcess.onExit (/home/runner/work/***/***/node_modules/node-gyp/lib/build.js:262:23)
npm ERR! gyp ERR! stack     at ChildProcess.emit (node:events:390:28)
npm ERR! gyp ERR! stack     at Process.ChildProcess._handle.onexit (node:internal/child_process:290:12)
npm ERR! gyp ERR! System Linux 5.4.0-1063-azure
npm ERR! gyp ERR! command "/usr/local/bin/node" "/home/runner/work/***/***/node_modules/node-gyp/bin/node-gyp.js" "rebuild" "--verbose" "--libsass_ext=" "--libsass_cflags=" "--libsass_ldflags=" "--libsass_library="
npm ERR! gyp ERR! cwd /home/runner/work/***/***/node_modules/node-sass
npm ERR! gyp ERR! node -v v16.13.0
npm ERR! gyp ERR! node-gyp -v v3.8.0
npm ERR! gyp ERR! not ok 
npm ERR! Build failed with error code: 1
</code></pre>
<p>My package.json is here:</p>
<pre><code class="lang-auto">{
  "name": "***",
  "version": "0.79.0",
  "private": true,
  "scripts": {
    "build": "node build/build.js",
    "dev": "webpack-dev-server --progress --config build/webpack.dev.conf.js",
    "lint": "eslint --ext .js,.vue src test/unit test/e2e/specs",
    "start": "npm run dev",
    "test": "npm run unit &amp;&amp; npm run e2e"
  },
  "browserslist": [
    "&gt; 1%",
    "last 2 versions",
    "not ie &lt;= 9"
  ],
  "resolutions": {
    "caniuse-lite": "^1.0.30001033",
    "growl": "^1.10.2"
  },
  "dependencies": {
    "@babel/runtime-corejs2": "7.10.2",
    "@popperjs/core": "2.10.1",
    "@riophae/vue-treeselect": "0.4.0",
    "axios": "0.21.3",
    "bootstrap-vue": "2.21.2",
    "deepmerge": "2.1.1",
    "email-regex": "4.0.0",
    "flag-icon-css": "3.5.0",
    "font-awesome": "4.7.0",
    "froala-editor": "4.0.4",
    "generate-password": "1.6.1",
    "jquery": "3.6.0",
    "js-cookie": "3.0.1",
    "ladda": "2.0.2",
    "liquor-tree": "0.2.70",
    "moment": "2.29.1",
    "moment-timezone": "0.5.33",
    "postcss-url": "8.0.0",
    "prettier": "2.3.2",
    "prettier-eslint": "9.0.1",
    "pretty-bytes": "5.6.0",
    "raw-loader": "0.5.1",
    "sanitize-html": "1.26.0",
    "simple-line-icons": "2.5.5",
    "speakingurl": "14.0.1",
    "sweetalert2": "10.3.5",
    "url-slug": "^2.3.2",
    "v-money": "0.8.1",
    "vee-validate": "2.2.12",
    "vue": "2.6.14",
    "vue-async-computed": "3.9.0",
    "vue-codemirror": "4.0.6",
    "vue-color-picker-wheel": "0.4.3",
    "vue-cropperjs": "4.2.0",
    "vue-currency-input": "1.18.4",
    "vue-froala-wysiwyg": "4.0.4",
    "vue-image-crop-upload": "3.0.3",
    "vue-loading-overlay": "3.4.2",
    "vue-monthly-picker": "0.2.8",
    "vue-multiselect": "2.1.6",
    "vue-numeric": "2.4.3",
    "vue-progressbar": "0.7.5",
    "vue-resource": "1.5.3",
    "vue-router": "3.5.2",
    "vue-select": "3.13.0",
    "vue-simple-context-menu": "3.4.1",
    "vue-smooth-dnd": "^0.8.1",
    "vue-snotify": "3.2.1",
    "vue-timeago": "^5.1.3",
    "vue2-datepicker": "3.10.1",
    "vue2-dropzone": "3.6.0",
    "vuejs-dialog": "1.4.2",
    "vuex": "3.6.2",
    "vuex-persistedstate": "3.0.1"
  },
  "devDependencies": {
    "autoprefixer": "7.2.5",
    "babel-core": "6.26.0",
    "babel-eslint": "8.2.2",
    "babel-helper-vue-jsx-merge-props": "^2.0.3",
    "babel-loader": "^7.1.4",
    "babel-plugin-istanbul": "^4.1.5",
    "babel-plugin-syntax-jsx": "^6.18.0",
    "babel-plugin-transform-runtime": "^6.23.0",
    "babel-plugin-transform-vue-jsx": "^3.5.0",
    "babel-preset-env": "^1.6.1",
    "babel-preset-stage-2": "^6.24.1",
    "babel-register": "^6.26.0",
    "chai": "^4.1.2",
    "chalk": "^2.3.2",
    "chromedriver": "^2.27.2",
    "connect-history-api-fallback": "1.5.0",
    "copy-webpack-plugin": "^4.5.1",
    "cross-env": "^5.0.1",
    "cross-spawn": "^5.0.1",
    "css-loader": "^0.28.11",
    "eslint": "5.16.0",
    "eslint-config-prettier": "^6.4.0",
    "eslint-config-standard": "^11.0.0",
    "eslint-friendly-formatter": "^4.0.0",
    "eslint-loader": "^2.0.0",
    "eslint-plugin-html": "^4.0.2",
    "eslint-plugin-import": "^2.9.0",
    "eslint-plugin-node": "^6.0.1",
    "eslint-plugin-prettier": "^3.1.1",
    "eslint-plugin-promise": "^3.7.0",
    "eslint-plugin-standard": "^3.0.1",
    "eslint-plugin-vue": "6.2.2",
    "extract-text-webpack-plugin": "^3.0.2",
    "file-loader": "^1.1.11",
    "friendly-errors-webpack-plugin": "^1.6.1",
    "html-webpack-plugin": "^2.30.1",
    "inject-loader": "^3.0.1",
    "karma": "^2.0.0",
    "karma-coverage": "^1.1.1",
    "karma-mocha": "^1.3.0",
    "karma-phantomjs-launcher": "^1.0.4",
    "karma-phantomjs-shim": "^1.5.0",
    "karma-sinon-chai": "^1.3.3",
    "karma-sourcemap-loader": "^0.3.7",
    "karma-spec-reporter": "^0.0.32",
    "karma-webpack": "^2.0.9",
    "less": "^3.0.4",
    "less-loader": "^4.1.0",
    "lolex": "2.3.2",
    "mocha": "^5.0.5",
    "nightwatch": "^0.9.20",
    "node-notifier": "^5.2.1",
    "sass": "1.44.0",
    "optimize-css-assets-webpack-plugin": "^3.2.0",
    "ora": "^1.2.0",
    "phantomjs-prebuilt": "^2.1.16",
    "portfinder": "^1.0.13",
    "postcss-import": "^11.1.0",
    "postcss-loader": "3",
    "rimraf": "^3.0.2",
    "sass-loader": "^6.0.7",
    "selenium-server": "^3.11.0",
    "semver": "^5.5.0",
    "sha1": "^1.1.1",
    "shelljs": "^0.8.1",
    "sinon": "^4.4.9",
    "sinon-chai": "^3.0.0",
    "string-replace-loader": "^2.3.0",
    "uglifyjs-webpack-plugin": "^1.2.4",
    "url-loader": "^1.0.1",
    "vue-loader": "^14.2.2",
    "vue-style-loader": "^4.1.0",
    "vue-template-compiler": "2.6.14",
    "webpack": "^3.11.0",
    "webpack-bundle-analyzer": "^2.11.1",
    "webpack-dev-server": "^2.11.2",
    "webpack-merge": "^4.1.2"
  },
  "engines": {
    "node": "&gt;= 8.9.4",
    "npm": "&gt;= 5.6.0"
  }
}
</code></pre>
<p>I changed the ubuntu version fro 18. I switch from node-sass to sass due to deprecations. I tried to install build-essentials before the install but it already got the latest version…</p>
<p>I have no idea where or what to fix? Can this be an error with Github Actions?</p>
<p>Thank you!</p>

**Comment by myd-mannheim** [comment]
<p>Hi, I had the same issue yesterday in my github action with cypress-io/github-action@v2. For me it helped to delete the package-lock.json and the node_modules folder. Then npm i (locally) and commit to git. Afterwards all errors were gone.</p>

**Comment by shroomlife** [comment]
<p>hey, thank you for your answer. The thing is I am not commiting the package-lock.json and also no node_modules folder. So on GitHub Actions run there is none of them present. I got no problem at all doing an npm install locally. Also my colleague got no issues on local. Also there were no changes to the package.json since months regarding something else than the version number…</p>

**Comment by shroomlife** [comment, marked as answer]
<p>I fixed the problem. I want to share my solution here.</p>
<p>I upgraded the sass-loader package as follows in package.json</p>
<pre><code class="lang-auto">    "sass-loader": "^6.0.7",
    "sass-loader": "^7",
</code></pre>

**Comment by ghost** [comment]
<p>my enterprise account? need the login I’ve forgot</p>
